docs/agent-reference/how-to-run-agent.md

```diff
@@ -8,12 +8,24 @@ AducIotAgent '<IoT device connection string>' [options...]
 
 ### Additional Arguments
 
-#### --EnableIotHubTracing
+#### --version
+
+With this option enabled, the reference agent will output the version.
+
+#### --enable-iothub-tracing
 
 With this option enabled, the reference agent will output verbose tracing from
 the Azure IoT C SDK. This option is useful for troubleshooting connection
 issues.
 
+#### --health-check
+
+With this option enabled, the reference agent will turn on Health Check Only mode.
```

Comment:
>Health Check Only mode [](start = 59, length = 22)

could we unpack this one a bit further? what is health check, why would a user want to turn it on.

Reply:
it's something that Nox added as part of selfupdate.  see my comment in this PR.

---
In reply to: [612718094](https://github.com/Azure/iot-hub-device-update/pull/39#discussion_r612718094) [](ancestors = 612718094)

Reply:
I checked that right now it is checking whether the Connection Info is valid; and whether the simulating unhealthy state (this is something we can set in aduc_conf file, but I am not sure about the usage). And there is a pending task for Nox on this, so I didn't include the description on simulating unhealthy state here

---
In reply to: [612719166](https://github.com/Azure/iot-hub-device-update/pull/39#discussion_r612719166) [](ancestors = 612719166,612718094)